.rubocop.yml

```diff
@@ -50,6 +50,9 @@ AccessModifierIndentation:
   Exclude:
     - 'lib/rspec/expectations/syntax.rb' # Too much diff to fix
 
+Layout/AlignParameters:
+  EnforcedStyle: with_first_parameter
```

Comment:
Can we enforce a style that leaves our preferred spacing? Then we can revert the rest of this commits changes.

Reply:
Honestly I think I would prefer this rule turned off then.

Reply:
Oh, just noticed that this setting is coming [from `rspec-dev`'s `.rubocop_rspec_base.yml`](https://github.com/rspec/rspec-dev/blob/master/travis/.rubocop_rspec_base.yml#L13).
Disabled this cop for this repository.